bin/sl-run.js

```diff
@@ -18,6 +18,30 @@ var agentOptions = {
   logger: config.logger,
 };
 
+var USE_PROBES = true;
+
+if (config.enableTracing) {
+  var tracer = require('../lib/tracer');
+  var traceObject = require('../lib/trace-object');
+  traceObject.tracerOptions.USE_PROBES = USE_PROBES;
+  traceObject.tracerOptions.transactions = true;
+  traceObject.tracerOptions.archiveInterval = 15000;
+  traceObject.tracerOptions.wrapRedis = true;
+  traceObject.tracerOptions.wrapMysql = true;
+  traceObject.tracerOptions.wrapMongodb = true;
+  traceObject.tracerOptions.wrapMemcached = true;
+  traceObject.tracerOptions.wrapPg = true;
+  tracer(traceObject.tracerOptions);
+}
+
+var coreTracer = tracer().tracer;
+agentOptions.strongTracer = coreTracer;
+agentOptions.wrapRedis = coreTracer.config.wrapRedis && USE_PROBES;
```

Comment:
strong-agent ***always*** wraps redis, whether wrapRedis is true or not.

It may not call transactionLink() for redis.

1. How much larger does json get when transactionLink() is called?
2. You realize that these "options" are not in fact options for our customers: they have no way to control the options, and `USE_PROBES` is hard-coded to true?